core/server/master/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -216,6 +216,14 @@ long getNewBlockIdForFile(AlluxioURI path) throws FileDoesNotExistException, Inv
    */
   Map<String, MountPointInfo>  getMountTable();
 
+  /**
+   * Gets the mount point information of an Alluxio path.
+   *
+   * @param path an Alluxio path
```

Comment:
add comment that path must be a mount point

Reply:
Added the comment, thanks!

Reply:
nit: combine comment with `@param path`?

Reply:
Moved to the description of `@param path`, thanks